k6StressTest.js

```diff
@@ -0,0 +1,36 @@
+import http from 'k6/http';
+import { sleep } from 'k6';
+import faker from 'https://cdnjs.cloudflare.com/ajax/libs/Faker/3.1.0/faker.min.js';
```

Comment:
why can't we import this as a normal package?

Reply:
import does not support node packages installed in app.